src/lib/swapper/swappers/CowSwapper/cowBuildTrade/cowBuildTrade.ts

```diff
@@ -123,8 +123,13 @@ export async function cowBuildTrade(
     rate,
     feeData: {
       networkFeeCryptoBaseUnit: '0', // no miner fee for CowSwap
-      buyAssetTradeFeeUsd: '0', // Trade fees for buy Asset are always 0 since trade fees are subtracted from sell asset
-      sellAssetTradeFeeUsd,
+      protocolFees: {
+        [sellAsset.assetId]: {
+          amountCryptoBaseUnit: feeAmountInSellTokenCryptoBaseUnit,
+          requiresBalance: false,
```

Comment:
Amazing. The code is pretty self-explanatory and acts as a documentation, but I might be biased since we've spiked on this one together - do we want to add a smolish comment explaining that CoW has no fees on the sell side, fees are implicit on the buy side and automagically subtracted as buy downside when swapping?